source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -263,32 +261,32 @@ internal void DrawDiagonalLine(Pen pen, int dx, int dy, int x1, int y1)
                         px += sdx;
                     }
                     py += sdy;
-                    DrawPoint(pen, px, py);
+                    DrawPoint(color, px, py);
                 }
             }
         }
 
         /// <summary>
         /// Draw line.
         /// </summary>
-        /// <param name="pen">Pen to draw with.</param>
+        /// <param name="color">color to draw with.</param>
         /// <param name="x1">Staring point X coordinate.</param>
         /// <param name="y1">Staring point Y coordinate.</param>
         /// <param name="x2">End point X coordinate.</param>
         /// <param name="y2">End point Y coordinate.</param>
         /// <exception cref="ArgumentOutOfRangeException">
         /// <list type="bullet">
-        /// <item>Thrown if pen is null.</item>
+        /// <item>Thrown if color is null.</item>
         /// <item>Coordinates invalid.</item>
         /// </list>
         /// </exception>
         /// <exception cref="Exception">Thrown on memory access violation.</exception>
         /// <exception cref="OverflowException">Thrown if x1-x2 or y1-y2 equal to Int32.MinValue.</exception>
-        public virtual void DrawLine(Pen pen, int x1, int y1, int x2, int y2)
+        public virtual void DrawLine(Color color, int x1, int y1, int x2, int y2)
         {
-            if (pen == null)
+            if (color == null)
```

Comment:
A value type can never be `null`; a null-check is unnecessary here (unless we're not using the `Color` type from `System.Drawing`)